new-client/src/models/layers/WMSLayer.js

```diff
@@ -37,6 +37,15 @@ class WMSLayer {
 
     overrideLayerSourceParams(source);
 
+    const minZoom =
+      config?.minZoom != null && config?.minZoom >= 0
+        ? config.minZoom
+        : undefined;
+    const maxZoom =
+      config?.maxZoom != null && config?.maxZoom >= 0
+        ? config.maxZoom
+        : undefined;
+
```

Comment:
`const minZoom = config?.minZoom >= 0 ? config.minZoom : undefined` should be enough?

Reply:
Correct. Will amend!